tsc/CMakeLists.txt

```diff
@@ -49,7 +52,12 @@ option(ENABLE_MRUBY "Enable the MRuby scripting engine" ON)
 option(ENABLE_NLS "Enable translations and localisations" ON)
 option(ENABLE_EDITOR "Enable the in-game editor" ON)
 option(USE_SYSTEM_TINYCLIPBOARD "Use the system's tinyclipboard library" OFF)
-option(USE_SYSTEM_MRUBY "Use the system's mruby (intended for internal use only!)" OFF)
+if(MSYS)
+  option(USE_SYSTEM_MRUBY "Use the system's mruby (intended for internal use only!)" ON)
```

Comment:
If possible, use TSC's included mruby. I'm not too fond that kirbyfan's flatpak doesn't use it either. I know I have to convince him at some point, but for now we should try to keep non-tsc mruby usage low.

Reply:
I have previously built TSC on msys2 with the built-in mRuby. But, as of now, the build in one complains during CMake Configure about prefixes being different `/c/something` vs `C:/something`, which msys2 handles just fine.

Reply:
> But, as of now, the built-in one complains during CMake Configure about prefixes being different

Sounds like a bug in our build setup. I suggest you leave it as it is for the moment and instead open an issue ticket about the problem, so that we won't forget about it.

Reply:
Uhm... I've got good news and not-so-good news. The good news is the builtin mRuby works! I tried  backtracking to replicate the error, but to no avail. I don't know what was wrong then, but now it works again...